docs/ProjectCommands.md

```diff
@@ -0,0 +1,64 @@
+# Project Command Reference
+
+Commands to do with projects. Remember that parameters with whitespace
+must be enclosed by quotation marks.
+
+## Options
+```sh
+/rocket project {list, view, help, create, unassign, edit, assign, delete}
+```
+
+### List
+```sh
+/rocket project list
+```
+
+Display a list of all projects.
+### View
+```sh
+/rocket project view PROJECT_ID
+```
+
+Displays details of project.
+
+### Help
+```sh
+/rocket project help
+```
+
+Displays options for `project` command.
+
+### Create (Team Lead and Admin only)
+```sh
+/rocket project create GH_REPO [--name DISPLAY_NAME]
```

Comment:
does this generate and return a project ID? how do we find PROJECT_ID, given a GH_REPO?

Reply:
This would generate and return one, yea.
Um, listing it would give both.

Reply:
listing could potentially show a large number of projects, unless we actively prune them (if every LP repo was a project, the `list` command would return 89 entries - in a Slack thread as well)

we'll probably need a query thing.

on that note, listing projects might need to be paged or something